### ISSUE TITLE
Issue with extension in example app

### ISSUE THREAD

**Issue by karptonite** [ngrx/platform#89]
Hey, trying to understand the new fractal state management, and ran into the following in the example app:
https://github.com/ngrx/platform/blob/master/example-app/app/books/reducers/index.ts#L22-L24
```typescript
export interface State extends fromBooks.State {
  'books': BooksState;
}
```
I don't entirely understand the new system yet, but to the degree that I do, I would have expected `State` to extend `fromRoot.State`, not `fromBooks.State`.  Am I confused, or is this an issue with the example app?

**Comment by brandonroberts** [comment]
Yes, this should be `fromRoot.State` instead of `fromBooks.State`

**Comment by karptonite** [comment]
Great, maybe I'm starting to understand this better than I thought!

**Comment by brandonroberts** [comment]
Closed via #90
